server_2x/src/api/legacy_runners/progress/state.js

```diff
@@ -1206,7 +1206,7 @@ class Runner {
                         farmPrinters[i]._id,
                     );
                     await farmPrinters[i].ws.open(
-                        `ws://${farmPrinters[i].printerURL}/sockjs/websocket`,
+                        `${farmPrinters[i].printerURL}/sockjs/websocket`,
```

Comment:
Hmm thought this haddn't updated but it's your legacy runner aha. Couldn't you just import these rather than keeping a carbon copy update? No idea like thought I'd ask aaha

Reply:
Those were meant to keep track of the old code and note down what still had to be done. A bad kind of progress tracker tbh, but better than nothing.
Can be removed if its in the way

Reply:
It's not in the way, just saves updating two sets of files was all I was thinking.